release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1150,6 +1150,15 @@ If you have Operator projects that were previously created or maintained with Op
 
 * xref:../operators/operator_sdk/java/osdk-java-updating-projects.adoc#osdk-upgrading-projects_osdk-java-updating-projects[Updating Java-based Operator projects]
 
+[discrete]
+[id="ocp-4-13-rhcos-symbolic-disk-naming-change"]
+=== It is now recommended to use persistent names for device parameters
```

Comment:
This heading is throwing me. I don't know that I've ever seen something like this. What about something like `Symbolic disk naming change for RHCOS based on {op-system-base} 9.2` or maybe `...with {op-system-base} 9.2}.

Reply:
this is a good suggestion! i've updated the heading to reflect this, thanks!